<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionDlg</class>
 <widget class="QDialog" name="SessionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Session manager</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QTreeWidget" name="sessionTree">
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="openSessionBtn">
     <property name="text">
      <string>Open session</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="removeSessionBtn">
     <property name="toolTip">
      <string>Remove selected session</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../juffed.qrc">
       <normaloff>:/trash.png</normaloff>:/trash.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="newSessionBtn">
     <property name="text">
      <string>New session</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../juffed.qrc"/>
 </resources>
 <connections/>
</ui>
